modules/nf-core/sentieon/dnascope/meta.yml

```diff
@@ -88,13 +88,11 @@ output:
   - index:
       type: file
       description: Index of the VCF file
-      pattern: "*.crai"
```

Comment:
Should the pattern be deleted or was that perhaps a mistake?

Reply:
Got it wrong at first since the index should be tbi and not crai 😄